src/services/discordBotServices.ts

```diff
@@ -6,8 +6,9 @@ export const updateUserRoles = async (env: env, payload: DiscordUserRole[]): Pro
 	try {
 		const url = config(env).DISCORD_BOT_API_URL;
 		const token = await generateDiscordBotJwt(env);
-
-		const response = await env.DISCORD_BOT.fetch(`${url}/roles?action=add-role`, {
+		//TODO(@Ajeyakrishna-k): remove dev flag https://github.com/Real-Dev-Squad/discord-slash-commands/issues/193
+		const devQuery = env.DEV ? '&dev=true' : '';
```

Comment:
What is `env.DEV`?

Reply:
its used as feature flag.

Reply:
Why just "DEV"? Why not something like "FEATURE_OWN_PRIVATE_KEY_... " etc

Reply:
changed it to `FF_SELF_PRIVATE_KEY`

Reply:
changed it to `FF_CRON_DISCORD_KEY_PAIR_FLOW`